tests/functional/browser.js

```diff
@@ -0,0 +1,73 @@
+const { expect } = chai;
```

Comment:
Do we need a separate test for browsers? Don't we have it already for node?

Reply:
We don't test the same file, on node we do not test the minified version ('dist/wallet-lib.min.js')

Reply:
I see. You don't have functional tests for node. Maybe we should run this test for node too?

Reply:
Done using the same basic logic (init wallet, get account, get balance, getUTXO and createTransaction). 
Having a balance not null is what actually tests most of the application as getting the balance requires having a proper working Worker + transporter + UTXO management in Storage etc...